packages/cli/src/lib/commands/collect/options/awaitServeStdout.ts

```diff
@@ -1,18 +1,8 @@
-import { argv } from 'yargs';
-import { Param } from './awaitServeStdout.model';
-import { ArgvOption } from '../../../core/yargs/types';
+import { Options } from 'yargs';
 
-// inspired by: https://github.com/GoogleChrome/lighthouse-ci/blob/main/packages/cli/src/collect/collect.js#L28-L98
-export const param: Param = {
-  awaitServeStdout: {
-    alias: 'w',
-    type: 'string',
-    description: 'A string in stdou resulting from serving the app, to be awaited before start running the tests. e.g. "server running..."',
-    implies: ['w', 'serveCommand']
-  }
-};
-
-export function get(): string {
-  const { awaitServeStdout } = argv as any as ArgvOption<Param>;
-  return awaitServeStdout;
-}
+export const awaitServeStdout = {
+  alias: 'w',
+  type: 'string',
+  description: 'A string in stdou resulting from serving the app, to be awaited before start running the tests. e.g. "server running..."',
+  implies: ['w', 'serveCommand']
+} satisfies Options;
```

Comment:
It seems strange and unnecessary to reap the option in an addition object called param.

Maybe it was used for the typing but this is not necessary and we do not need to do this. 

It also means we would need to have a bunch of extra code to infer typing!